resources/hvac_sizing.rb

```diff
@@ -2328,17 +2328,17 @@ def self.get_hvacs(runner, model)
     hvacs = []
 
     # Get unique set of HVAC equipment
-    equips = {}
+    equips = []
 
     HVAC.existing_equipment(model, runner, @cond_zone).each do |equip|
-      next if equips.keys.include? equip
+      next if equips.include? equip
       next if equip.is_a? OpenStudio::Model::ZoneHVACIdealLoadsAirSystem
 
-      equips[equip] = @cond_zone
+      equips << equip
```

Comment:
Found one more example of code that handled multiple conditioned zones. Thanks for all that cleanup you did on that!